drivers/gpu/drm/amd/amdgpu/amdgpu_drv.c

```diff
@@ -36,6 +36,7 @@
 
 #include <drm/drm_pciids.h>
 #include <linux/console.h>
+#include <linux/dmi.h>
```

Comment:
in the commit message please briefly mention which tests you have done on this unit